src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -27,16 +27,17 @@ class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter) extends
 
   private val subscribedEventTypes = request.getParameterMap.getOrDefault("event_type", Array.empty).toSet
 
-  def subscribed(eventType: String): Boolean = {
-    subscribedEventTypes.isEmpty || subscribedEventTypes.contains(eventType)
-  }
+  def subscribedForEventType(eventType: String): Boolean = subscribedEventTypes.isEmpty || subscribedEventTypes.contains(eventType)
+  def subscribedForTaskId(taskId: String): Boolean       = subscribedEventTypes.isEmpty || subscribedEventTypes.contains(task_id)
```

Comment:
I believe a newly introduced `subscribedForTaskIds` variable should be used here instead. Also, this code fails to compile, because there is no `task_id` variable.

Reply:
In fact, the problem is, there's no mention of Task in any way in events except for UnhealthyInstanceKillEvent. At this stage, we know event type and it's timestamp. Is there a way to look up TaskId based on those inputs?